### PR TITLE
Add new i3 class instances.

### PR DESCRIPTION
**Pull request by rogersd**

### CODE REVIEW

**Comment by lanej** [comment]
From https://aws.amazon.com/ec2/instance-types/

image

File: lib/fog/aws/models/compute/flavors.rb [hunk 1]
```diff
@@ -425,6 +425,66 @@ class AWS
...
+          :ebs_optimized_available => true,
+          :instance_store_volumes  => 4
+        },
+        {
+          :id                      => 'i3.16xlarge',
+          :name                    => 'I3 Sixteen Extra Large',
+          :bits                    => 64,
+          :cores                   => 32,
+          :disk                    => 15200,
+          :ram                     => 249856,
```
Comment (lanej) [inline lib/fog/aws/models/compute/flavors.rb]: This should be `(488 * 1024) = 499712`.

Comment (rogersd) [inline lib/fog/aws/models/compute/flavors.rb]: Curse you, copy and paste!

**Comment by rogersd** [comment]
Fixed.

**Comment by geemus** [comment]
Thanks!

**Pull request merged**